src/main/java/com/ericsson/ei/handlers/RmqHandler.java

```diff
@@ -150,6 +162,26 @@ public ConnectionFactory connectionFactory() {
         return cachingConnectionFactory;
     }
 
+    @Bean
+    public RabbitManagementTemplate deleteBindingKeys() {
+    	Integer rabbitPort=port;
+    	String rabbitHttp = "https://";
+    	if(port==5672 && host.equals("localhost")){
```

Comment:
What is the logic behind this code block? The API is exposed through the management page usually located at 15672 but if the port is anything other than 5672 we don't attempt to redirect to the website API? Won't this fail unless the user specifially configures the API url?

Reply:
Hi,Actually for removing old binding keys we have gone through some websites on google, and found out the RabbitManagementTemplate to remove the old binding keys and used for implementation. So as part of it we are facing issues with RabbitManagementTemplate in test class. So we tried to solve it with this kind of approach, just for testing purpose.